tests/scripts/save_image_transforms.py

```diff
@@ -0,0 +1,69 @@
+from pathlib import Path
```

Comment:
Could we rename `save_image_transforms_to_safetensors.py` for consistency?

Reply:
Fixed https://github.com/huggingface/lerobot/pull/234/commits/1890637f27c5a917f54dedf70fc6fad9b60a6ee5